src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/FishboneMoncriefDisk.hpp

```diff
@@ -44,10 +44,15 @@ namespace Solutions {
  * where \f$u^\mu\f$ is the 4-velocity. Then, all the fluid quantities are
  * assumed to share the same symmetries as those of the background spacetime,
  * namely they are stationary (independent of \f$t\f$), and axially symmetric
- * (independent of \f$\phi\f$). Self-gravity is neglected, so that the fluid
+ * (independent of \f$\phi\f$). Note that \f$r\f$ and \f$\theta\f$ can also be
+ * interpreted as Kerr (a.k.a. "spherical" Kerr-Schild) coordinates
+ * (see KerrSchildCoords), and that the symmetries of the equilibrium ensure
```

Comment:
Based on the wording, I believe "KerrSchildCoords" was intended to be a link.  For some reason Doxygen is not generating the link automatically, but adding the `gr::` namespace prefix convinces it to do so.